cli/command/system/info.go

```diff
@@ -318,14 +322,7 @@ func prettyPrintServerInfo(dockerCli command.Cli, info types.Info) []error {
 	fprintlnNonEmpty(dockerCli.Out(), " HTTP Proxy:", info.HTTPProxy)
 	fprintlnNonEmpty(dockerCli.Out(), " HTTPS Proxy:", info.HTTPSProxy)
 	fprintlnNonEmpty(dockerCli.Out(), " No Proxy:", info.NoProxy)
-
-	if info.IndexServerAddress != "" {
-		u := dockerCli.ConfigFile().AuthConfigs[info.IndexServerAddress].Username
-		if len(u) > 0 {
-			fmt.Fprintln(dockerCli.Out(), " Username:", u)
-		}
-	}
-
+	fprintlnNonEmpty(dockerCli.Out(), " Username:", info.UserName)
```

Comment:
FWIW, I'm also very much leaning towards moving this information to the `Client:` section, because nothing about it is part of the "Server:".

I _think_ this was historically because it _depended_ on the server (?) but no real idea.

Also;

- because it's not part of the actual `info.Info` struct (i.e., the `Server` part), it's currently not even possible to use `docker info --format <template>` to show the name
- ^^ so `docker info --format=json` won't show it
- ^^ _AND_ if a credentials-helper is used, it's not even filled in _at all_ (it only looks in the config file
- 🙃 🤷

Reply:
So ... perhaps we should either remove it altogether, or make it use the actual credential-store in use, BUT reading info from that is not always .. fast, and I don't want `docker info` to become slow 😓